polylogue/commands.py

```
@@ -635,58 +641,68 @@ def sync_command(options: SyncOptions, env: CommandEnv) -> SyncResult:
+                totals_acc.increment("skipped")
+                tracker.advance()
+                continue
+
+            if not options.dry_run and chat_context.modified_time:
+                mtime = parse_rfc3339_to_epoch(chat_context.modified_time)
+                if mtime is not None:
                     try:
-                        os.utime(result.html_path, (mtime, mtime))
+                        os.utime(result.markdown_path, (mtime, mtime))
                     except Exception:
                         pass
```

Ruff (0.15.9)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)